src/abstractdataframe/abstractdataframe.jl

```diff
@@ -608,6 +608,9 @@ end
 completecases(df::AbstractDataFrame, cols::AbstractVector) =
     completecases(df[cols])
 
+completecases(df::AbstractDataFrame, cols::Regex) =
```

Comment:
Could just use `::Union` in the previous method.

Reply:
fixed